NGS2_WITNESS_Cycle1_confirmatory_exp1.R

```diff
@@ -0,0 +1,73 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. in March 2017
+
+# Load data for analysis
+exp1_cooperation <- read.csv('NGS2-Cycle1-Experiment1/cooperation_exp1.csv',
+                             header = TRUE, sep = ',')
+exp1_rewire <- read.csv('NGS2-Cycle1-Experiment1/rewire_exp1.csv', header = TRUE,
+                        sep = ',')
+
+### Show stopper
+
+print("ATENTION!! Need to make sure the exp1_cooperation dataset includes the following variables: sessionnum, condition, decision..0.D.1.C.")
+
+#Hypothesis 4.1.1: Fixed network compositions reduce exp1_cooperation
+#Rand  et al. (2011) coeff = -0.186, p<.000
+
+exp1_cooperation.fixed<-subset(exp1_cooperation, condition=="Static")
+logit.fixed <- glm(decision..0.D.1.C.~ round_num, data = exp1_cooperation.fixed, family = "binomial")
+logit.multiwayvcov.fixed <- cluster.vcov(logit.fixed, cbind(exp1_cooperation.fixed$sessionnum, exp1_cooperation.fixed$playerid))
+coeftest(logit.fixed, logit.multiwayvcov.fixed)
+
+#Hypothesis 4.1.2: Randomly updating network compositions reduce exp1_cooperation
+#Rand  et al. (2011) coeff = -0.113, p<.000
+
+exp1_cooperation.random<-subset(exp1_cooperation, condition=="Random")
+logit.random <- glm(decision..0.D.1.C.~ round_num, data = exp1_cooperation.random, family = "binomial")
+logit.multiwayvcov.random <- cluster.vcov(logit.random, cbind(exp1_cooperation.random$sessionnum, exp1_cooperation.random$playerid))
+coeftest(logit.random, logit.multiwayvcov.random)

```

Comment:
With the `coeftest()`, irrespective of the function described above, this I don't think is being saved out anywhere. If this is pipelined, don't we want to save these results somewhere? Shouldn't the results from the various `coeftest()` calls be saved for inspection/use later on?